src/components/Character/styles.scss

```diff
@@ -1,19 +1,189 @@
+	padding: 10%; 		//GUESS
+	flex-direction: column;
 
 	&[aria-hidden="true"]{
 		opacity: 0;
+		pointer-events: none
 	}
 }
-	.button {
-		&.next {
-			font-size: 1em;
+
+	.container {
+		position: relative;
+		z-index: 1;
+	}
+
+		.content {
+			position: relative;
+			margin-top: 40vh;
+		}
+	
+			.name {
+				margin-bottom: 0.1em; // GUESS
+				font-size: 5em;
+				font-family: var(--font_family__heading);
+			}
+
+			.nickname {
+				margin-bottom: 0.5em; // GUESS
+				font-size: 2.6em;
+				font-family: var(--font_family__heading);
+			}
+
+			.actor {
+				margin-bottom: 1em; // GUESS
+				font-size: 1em;
+				font-family: var(--font_family__body);
+				font-weight: normal;
+				letter-spacing: 0.25em;
+				text-transform: uppercase;
+			}
+
+			.body {
+				margin-bottom: 4em; // GUESS
+				line-height: 1.5;
+				font-family: var(--font_family__body);
+			}
+
+		.content::after {
+			content: "";
+			display: block;
+			position: fixed;
+			width: 100%;
+			height: 12em;
+			bottom: 0;
+			left: 0;
+			background-image: linear-gradient(
+			    rgba(237, 20, 61, 0) 0%,
+			    rgba(237, 20, 61, 1) 50%,
+			    rgba(237, 20, 61, 1) 100%	
+			);
+		}
+
+		.controls {
+			display: flex;
+			position: sticky;
+			width: 100%;
+			max-width: 25rem; // GUESS
+			bottom: 10vw; // GUESS!
+			justify-content: space-between;
+			font-size: 1.1em;
+			font-family: var(--font_family__heading);
 		}
-		&.previous {
-			font-size: 1em;
+
+			.button {
+				display: flex;
+				align-items: center;
+				text-decoration: none;
+				--button_margin: 1em;
+
+				&::before, 
+				&::after {
+					display: inline-block;
+					font-size: 1.36em;
+					width: 1em;
+					height: 1.5em;
+					background-color: lime;
+				}
+
+				&.previous {
+					&::before {
+						content: "";
+						margin-right: var(--button_margin);
+					}
+				}
+
+				&.next {
+					&::after {
+						content: "";
+						margin-left: var(--button_margin);
+					}
+				}
+			}
+
+// Mobile landscape
+@media (orientation: landscape) {
+	.wrapper {
+		max-height: 100%;
+		padding: 5%; //GUESS
+	}
+
+		.container {
+			width: 50%; 
+			height: 100%;
+			--button__bottom_spacing: 5vw;
 		}
+
+			.content {
+				margin-top: 0;
+			}
+
+				.body {
+					height: 25vh; //GUESS
+					margin-bottom: 2em; //GUESS
+					overflow-y: scroll;
+				}
+
+			.controls {
+				bottom: var(--button__bottom_spacing); //GUESS
+			}
+
+			.content::after {
+				position: fixed;
+				bottom: var(--button__bottom_spacing); //GUESS
+				height: 6em; //GUESS
+				pointer-events: none;
+				background-image: linear-gradient(
+				    rgba(237, 20, 61, 0) 0%,
+				    rgba(237, 20, 61, 1) 50%,
+				    rgba(237, 20, 61, 1) 100%	
+				);
+			}
+}
+
+// Tablet portrait
+@media (min-width: 768px) {
+	.wrapper {
+		padding: 10%;
 	}
+}
+
+// Tablet landscape
+@media (min-width: 768px) and (orientation: landscape) {
+	.wrapper {
+		justify-content: center;
+	}
+
+		.container {
+			--button__bottom_spacing: 15vw; //GUESS
```

Comment:
yeah, this is an indication of something going wrong - it's a clever use of CSS variables, but it's still essentially a magic number; it would be better to use a solution like the `position: absolute` one mentioned above that doesn't break it out of flow - currently this looks a bit broken:
![image](https://user-images.githubusercontent.com/3370135/65994692-dc5a4000-e48b-11e9-8b61-99621a8a387e.png)
vs
![image](https://user-images.githubusercontent.com/3370135/65994719-eb40f280-e48b-11e9-99f5-a46f2d514f3b.png)